easybuild/easyconfigs/r/rMATS-long/rMATS-long-1.0.0-foss-2023a-R-4.3.2.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'Tarball'
+
+name = 'rMATS-long'
+version = '1.0.0'
```

Comment:
If we take a more recent commit that the `1.0.0` tag, we should also add a datestamp to the version:
```
version = '1.0.0-20240502'
```

Reply:
fixed in ef85e1f